### ISSUE TITLE
App doesn't show the moderators in the side bar

### ISSUE THREAD

**Issue by sallyFoster** [aeharding/voyager#967]
**Describe the bug**

The app doesn't show the moderators in the side bar

**To Reproduce**

Go to sidebar on any community and scroll down to the Moderators section

**Expected behavior**
A list of the moderators should show. 

**Screenshots**
![image](https://github.com/aeharding/voyager/assets/104597302/d796b9ff-9cac-4b3d-a19f-ed45d12c23fe)

**Smartphone (please complete the following information):**

- Device: iPhone 11 Pro Max
- OS: iOS 17.1

**Comment by aeharding** [comment]
Hi there! I cannot reproduce, on my devices Voyager is showing the mods of communities.

If you can provide any additional detail to help reproduce, that would be helpful. Thanks!

**Comment by sallyFoster** [comment]
@aeharding It intermediately works and then it doesn’t work 

https://github.com/aeharding/voyager/assets/104597302/5857e2bf-5874-4c0c-a1d2-2c632b4a71fc

**Comment by aeharding** [comment]
Hi @sallyFoster 

The exact steps for how you reproduce may help. I found a bug through some investigation where searching for a community, not tapping it, then going to home/all/local feed and finding that community and bringing up the community sidebar through a long press on the community label (if the community was found in the search) will cause the mod list to be empty, so I have a fix for that.

That's the only edge case I was able to find, so hopefully this fixes your issue!
